hw/top_earlgrey/rdc/chip_earlgrey_asic_scenario.tcl

```diff
@@ -14,36 +14,87 @@
 # POR_N
 # When POR_N released, POK remains high (already released from reset)
 # dio_pad_attr_q[12] is SCK PAD. The invert signal remains 0 always
+#
+# TODO: Create two scns (either vcmain_pok 0 (turn off all clock), or
+#       vcmain_pok 1 (all clocks alive))
 set_reset_scenario { \
   { POR_N           { reset {#2 0} { #10 1} }} \
-  { u_ast.vcaon_pok { constraint {@t0 1} }} \
+  { u_ast.u_rglts_pdm_3p3v.vcaon_pok_h  { constraint {@t0 1} }} \
   { top_earlgrey.u_pinmux_aon.dio_pad_attr_q[12].invert \
     { constraint { @t0 0 } } } \
   { top_earlgrey.u_spi_device.cio_sck_i { constraint { @t0 0 } } } \
 } -name ScnPOR
 
-# AST POK
+# AST AON POK
```

Comment:
I will have to take a closer look at the ones below - will comment in a bit.

Reply:
The scenarios LGTM, although I was wondering whether we should ask Nuvoton to look these over also? In essence, we are capturing AST behavior here, right?

@tjaychen WDYT?

Reply:
yeah @eunchan and i were discussing earlier, it looks good to me.
Maybe we can just confirm with Nuvoton / Ziv via email? Probably don't need them to look through all of this...

Reply:
Yep. I added this based on Tim's suggestion. Thanks Tim!